src/telemetry/telemetry.c

```diff
@@ -231,7 +230,7 @@ get_num_drop_chunks_policies()
 {
 	StringInfo buf = makeStringInfo();
 
-	appendStringInfo(buf, "%d", ts_bgw_policy_drop_chunks_count());
+	appendStringInfo(buf, "%d", ts_bgw_policy_retention_count());
```

Comment:
Is it still compatible?

Reply:
i dont change the name in the telemetry json need to talk to solar if that would break stuff